test/test_complex_aliases.py

```diff
@@ -0,0 +1,60 @@
+import sqlparse
+
+from sql_metadata import Parser
+
+import pathlib
+
+dir_path = pathlib.Path(__file__).parent.absolute()
+
+
+def test_complex_query_aliases():
+    sql_filename = f"{dir_path}/test.sql"
+    with open(sql_filename, "r", encoding="latin-1") as content_file:
+        content = content_file.read()
+    content = sqlparse.format(content, strip_comments=True).strip()
```

Comment:
Why is this step necessary?

Reply:
Not sure, took it directly from PR #99, I guess it should work without it - will check

Reply:
BTW. maybe we should use sqlparse functionality instead of regex to remove comments if it already exist?

Reply:
Confirm it works just fine without the sqlparse line

Reply:
That's a good idea - query pre-parsing step 👌🏻

Reply:
Removed extra step.